example/email_action_opened.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
@josephpconley general question about these email events... are we not sending over the URL to the market/buyer report?

Reply:
@dduran1967 does it make sense to add the url as follows?

```
"messageAttachment": [
      {
        "type": "DigitalDocument",
        "id": "http://stage-valuations-foxroach-com.hvs.buyermls.com/report/#/25354/49461/market-activity-report/430+Foulke+Ln/Springfield/PA/19064"
      }
    ]
```

Reply:
@josephpconley - Yes, that would work if there is an attachment.  You can add it to the example and I'll get it into the schema / docs

Reply:
@dduran1967 it's not technically an attachment though, but a link within the message, is that OK?

Reply:
that's fine, let's discuss on the call